app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_, file_to_move, path_to_move = command.split()
```

Comment:
Make initial checks that the command contains `mv` and consists of three elements. Besides that, the second and the third arguments should not be the same. Place all checks in one if condition.